tinywallet/tinywallet/app.py

```diff
@@ -563,7 +566,7 @@ def runTinyDecred():
     qApp.setApplicationName("Tiny Decred")
     loadFonts()
 
-    decred = TinyDecred(qApp)
+    decred = TinyDecred(qApp, cfg)
```

Comment:
It doesn't actually look like the `cfg` is being used before `TinyDecred` is created. It can just be created in the constructor if so.

Reply:
Good point, moving it there.

Reply:
Oh, and the `log` handler doesn't have to be a global either, it can be a `TinyDecred` attribute instead. Then the `tryExecute` function can become a method of that class, so that it can access the log handler without having to pass it as a parameter.

Reply:
`tryExecute` is only used once, and it's pretty sloppy anyway, so feel free to rename or redefine it any way that fits.